cfpb/capital-framework
# Use brand colors in Capital Framework documentation, add cf-notification

Remove `cf-theme-overrides.less` and `brand-palette.less` from docs, use brand colors from updated components in release 4.0.9

## Additions

- cf-notification dependency for compiling docs

## Removals

- deleted brand-palette.less and cf-theme-overrides.less. These variables are now defined in cf-core and individual components only. 

## Changes

- Compiled new docs

## Testing

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- run `npm run build`
- run `npm start`
- visit http://localhost:3000/components/ to browse updated docs

## Review

- @anselmbradford 

## Screenshots

buttons and typography with some brand colors

![screen shot 2017-09-06 at 12 48 16 pm](https://user-images.githubusercontent.com/702526/30124129-3e241fa0-9302-11e7-8142-481821446a7b.png)
![screen shot 2017-09-06 at 12 47 28 pm](https://user-images.githubusercontent.com/702526/30124128-3e2413e8-9302-11e7-98b9-52d975ff1cb5.png)


## Notes

-

## Todos

-

## Checklist
